WorkAndFun/src/main/java/pl/infoshare/workandfun/announcements/announcement_repo/entity/Announcement.java

```diff
@@ -24,11 +25,17 @@
     private Long id;
     @Enumerated(EnumType.STRING)
     private Type type;
+    @NotEmpty
+    @Size(min = 10,max = 60)
     private String header;
     @Enumerated(EnumType.STRING)
     private ServiceType serviceType;
```

Comment:
Trzeba by się pochylić nad wyborem serviceType, ponieważ aktualnie wybór jest na zasadzie wszystko w jednym, a powinno być na zasadzie drzewa. Wiem, że nie było nic takiego na zajęciach, jednak fajnie by było to ogarnąć. Można napisać do jakiegos trenera z prośbą o wsparcie. Nie czepiam się tego, jedynie zostawiam moją sugestię i proponuje o dopisanie TODO